draft-ietf-quic-transport.md

```diff
@@ -1674,6 +1675,12 @@ disable_migration (0x0009):
   address other than that used to perform the handshake.  This parameter is a
   zero-length value.
 
+max_ack_delay (0x000c):
+
+: An 8 bit unsigned integer value indicating the maximum amount of time in
```

Comment:
duration?

Reply:
How about, "the maximum ACK delay in milliseconds"?

Reply:
I think it's helpful to clarify what ACK delay is here, so I'd like to avoid using the same words in the parameter when possible.

Reply:
I agree that it would be helpful. The only reference to ACK Delay in the transport draft is in the description of the ACK frame. The ACK generation is described in the recovery draft, using a parameter kDelayedAckTimeout. Max ACK Delay is also defined in the recovery draft, but the section 4.1.1 appears obsolete. In light of that, the proper phrase is probably "maximum time that the peer will wait before acknowledging a packet" plus a reference to the appropriate section of the recovery draft. This PR should probably also update the recovery draft.